### PR TITLE
Contact form validation and design improvements

### PR DESCRIPTION
**Pull request by MuhammadUsmanAslam**
- Contact form validation for their respective input types which will check if the required values are entered with their correct types.
- Design improvements for the contact form to make it more user-friendly and look better.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -206,13 +206,14 @@ <h1 class="contact-me-form-heading">Contact me</h1>
...
-                placeholder="Your email address">
-            <textarea class="contact-me-form-text-input" name="" id="" cols="30" rows="9"
-                placeholder="Write your message here"></textarea>
-            <button class="button">Get In Touch</button>
+        <form action="https://formspree.io/f/mqkoaqjr" method="POST" class="contact-me-form">
+            <input class="contact-me-form-text-input" required maxlength="30" name="name" type="text" value="Usman"
+                placeholder="Your good name">
+            <input class="contact-me-form-text-input" required name="email" type="email"
+                value="musmanaslam520@gmail.com" placeholder="Your email address">
+            <textarea class="contact-me-form-text-input" required maxlength="500" name="message" id="" cols="30"
```
Comment (Zuhaib042) [inline index.html]: - [OPTIONAL] kindly consider removing `id=""` as you are not giving any id to `textarea` element as it is against DRY principle.

File: style.css [hunk 2]
```diff
@@ -168,7 +168,7 @@ button {
   font-family: inherit;
   background: #fff;
   color: #396df2;
-  border: 1px solid #396df2;
+  border: 1px solid #fff;
   border-radius: 0.5rem;
   padding: 0.75em;
   font-size: 1.05em;
```
Comment (Zuhaib042) [inline style.css]: - [x] Good job styling the button but there are some issues you need to fix in order to get your project approved. kindly consider applying the button interactions in :hover and :active states, and we must apply those colors shown in the Figma design in those states.
I can see that these interactions have not been applied to your other buttons as well in your project. kindly consider applying these button interactions. I am uploading Figma design picture to get the appropriate colors for different states. kindly consider applying these same colors.
  - Button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template.
asd

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @MuhammadUsmanAslam  @shurichmond , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
close

## Highlights
✅ Correct Github flow
✅ Linters passing
✅ Meaningful commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ OPTIONAL] Kindly consider adding a Descriptive title to your pull request describing what changes are inside.
- [OPTIONAL] Also I will highly recommend adding bullet points in your pull request summary under the title instead of writing sentences in a paragraph, that would make your pull request much more professional.
- [OPTIONAL] Kindly consider removing the Microverse logo from the readme as it is unprofessional. you can use your own logo in your readme if you have one.
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @MuhammadUsmanAslam , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

- [ ] As requested by the previous reviewer, Why don't you consider removing the Microverse logo from the readme as it is unprofessional? you can use your own logo in your readme if you have one.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**